src/vmm/src/version_map.rs

```diff
@@ -76,6 +75,8 @@ lazy_static! {
         // between firecracker minor versions and snapshot versions needs
         // to be 1-to-1 (see below)
         version_map.new_version();
+        #[cfg(target_arch = "aarch64")]
+        version_map.set_type_version(VcpuState::type_id(), 2);
```

Comment:
nit: line 74 `v1.5 - no changes introduced` is no longer true?

Reply:
fixed